napari/layers/surface/surface.py

```diff
@@ -342,6 +351,8 @@ def _get_state(self):
                 'gamma': self.gamma,
                 'shading': self.shading,
                 'data': self.data,
+                'wireframe': self.wireframe.to_dict(),
```

Comment:
Getting fails

```python-traceback
  E       AttributeError: 'SurfaceWireframe' object has no attribute 'to_dict'
```

Should this just be `self.wireframe.dict()`? I can't remember off top of my head

Reply:
Fixed, which however made something else clear: this cannot work since `wireframe` is no longer an argument of `points`. should it still be an argument, and accept `dict | SurfaceWireframe` as types?

Reply:
Yep that sounds good! It's what we did for Slicing plane and co 🙂

Reply:
Yeah that's where I looked for inspiration :)